SpareMe/ios/SpareMe.xcodeproj/project.pbxproj

```diff
@@ -42,7 +42,12 @@
 		832341BD1AAA6AB300B99B32 /* libRCTText.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 832341B51AAA6A8300B99B32 /* libRCTText.a */; };
 		AB988DFCB49A26D00B338687 /* libPods-SpareMe-tvOS.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 1E721F34E09B5FBE3C631E16 /* libPods-SpareMe-tvOS.a */; };
 		ADBDB9381DFEBF1600ED6528 /* libRCTBlob.a in Frameworks */ = {isa = PBXBuildFile; fileRef = ADBDB9271DFEBF0700ED6528 /* libRCTBlob.a */; };
+<<<<<<< HEAD
```

Comment:
I'm blocked from reviewing this until these are resolved because I can't open the Xcode project